ts/application.ts

```diff
@@ -1,6 +1,5 @@
 import { EventEmitter } from 'events'
-import { Libp2pCryptoIdentity } from '@textile/threads-core'
```

Comment:
`Libp2pCryptoIdentity` has been replaced with the more useful `PrivateKey`, `PublicKey`, and `Identity` classes